### PR TITLE
Working blog

### PR DESCRIPTION
**Pull request by awojciak**

### CODE REVIEW

File: src/components/Blog/Article/Article.tsx [hunk 1]
```diff
@@ -1,19 +1,29 @@
 import { CardMedia, Grid, Typography } from '@material-ui/core'
 import React from 'react'
 import { DataForArticles } from '@components/Blog/Blog'
+import Link from 'next/link'
 import useStyles from './style'
 
-export const Article: React.FC<DataForArticles> = ({ title, date, image }) => {
+export const Article: React.FC<DataForArticles> = ({ title, date, image, slug, description }) => {
   const classes = useStyles()
   console.log(image)
```
Comment (Kimi82) [inline src/components/Blog/Article/Article.tsx:9]: Delete comments

File: src/blogPosts/hello-world.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: "Hello world!"
+date: "2021-09-16"
```
Comment (PawelKrawczyk96) [inline src/blogPosts/hello-world.md:3]: Invalid date format (compare to figma project)

**Review by PawelKrawczyk96 — COMMENTED** [review]
Text at the bottom must be on the one line
Screenshot from 2021-09-16 12-28-00

File: src/blogPosts/hello-world.md (hunk 2 above)
Comment (awojciak) [reply src/blogPosts/hello-world.md:3]: fixed

File: src/components/Blog/Article/Article.tsx (hunk 1 above)
Comment (awojciak) [reply src/components/Blog/Article/Article.tsx]: fixed

**Review by Kimi82 — APPROVED** [review]

**Pull request merged**